lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
```

Comment:
Possibly, but instances can override electron versions at construction time.  Maybe I'm not understanding what you're getting at?